data-sources/atlassian-jira/config.yml

```diff
@@ -3,10 +3,13 @@ displayName: Atlassian JIRA
 description: |
   Jira is a proprietary issue tracking product developed by Atlassian that allows bug tracking and agile project management. You can Integrate New Relic with Atlassian Jira(Cloud) and automatically create and update Jira issues.
 icon: logo.png
-install:
+install:  
   primary:
-    link:
-      url: https://docs.newrelic.com/docs/alerts-applied-intelligence/notifications/notification-integrations/#jira
+    nerdlet:
+      nerdletId: marketplace.install-data-source
+      nerdletState:
+        dataSourceId: atlassian-jira
```

Comment:
I have observed that the "data source ID" named "atlassian-jira" is not attached to any quickstart. Could you please attach it to the related quickstart to proceed further.

Reply:
updated the code to fix the suggestions